src/styles/header_moyo/headerStyle.scss

```diff
@@ -0,0 +1,79 @@
+ul,
+li,
+body,
+nav {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
same here, you should move this into `main.scss` or create another separate file to reset